front/src/component/mypage/ProfileDetail.js

```diff
@@ -24,11 +32,11 @@ export default ({ user }) => {
         {/*    </button>*/}
         {/*  </div>*/}
         {/*</div>*/}
-        {/*<div className="withdrawal">*/}
-        {/*  <a href="#" onClick={() => alert("hi")} className="withdrawal-submit">*/}
-        {/*    회원 탈퇴*/}
-        {/*  </a>*/}
-        {/*</div>*/}
+        <div className="withdrawal">
+          <a href="/" onClick={userWithdraw} className="withdrawal-submit">
+            회원 탈퇴
+          </a>
```

Comment:
Link 태그를 사용해도 좋을 것 같아요!

```suggestion
          <Link to="/" onClick={userWithdraw} className="withdrawal-submit">
            회원 탈퇴
          </Link>
```

IDE 상에서 한게 아니고 깃헙 댓글 작성으로 수정한 부분이라 정확하지 않을 수 있습니다.